### PR TITLE
RS reorg: access control

### PR DESCRIPTION
**Pull request by rrelledge**
Here's a preview of the initial reorg of the access control docs.

[Staged preview](https://docs.redis.com/staging/reorg-rs-rbac/rs/security/access-control/)

### CODE REVIEW

**Review by kaitlynmichael — APPROVED** [review]
LGTM

File: content/rs/security/access-control/_index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: An overview of access control in Redis Enterprise Software.
+weight: 10
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/administering/access-control/user-roles",
+          "/rs/security/passwords-users-roles/"]
+---
+Role-based access control allows you to scale your Redis deployments while minimizing the overhead involved in managing a cluster with many databases, multiple users, and various access control lists. With RBAC, you can create a role once and then deploy it across multiple databases in the cluster.
+
+You can configure roles with standard or custom templates for database permissions that are based on the Redis ACL syntax. Redis Enterprise allows you to restrict database operations by command, command category, and key pattern.
```
Comment (AlonMagrafta) [inline content/rs/security/access-control/_index.md:13]: We can also do specific pub.sub channels (same as OSS).
But the general statement is correct, so maybe I should not be so needy on the smaller details

File: content/rs/security/access-control/active-active.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+weight: 60
+alwaysopen: false
+categories: ["RS"]
+aliases: 
+---
+
+Users, roles, and Redis ACLs are cluster-level entities. Therefore they are applied per local participating cluster and Active-Active database instance, and they are not replicated or propagated to the other participating clusters and instances.
+ACLs will be enforced according to the instance the client is connected to. The Active-Active replication mechanism will propagate all the effects of the operation.
+
+## Update admin credentials
```
Comment (AlonMagrafta) [inline content/rs/security/access-control/active-active.md:14]: This page holds two things:
1. The above - ACL - is about data access permissions.
2. The below - admin user - is about the control plane Admin user.
I think we need to clarify it a little bit more, maybe a title for the first will do the work. They just seems connected to me visually at the moment

File: content/rs/security/access-control/manage-users.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
+---
+Title: Manage users
```
Comment (AlonMagrafta) [inline content/rs/security/access-control/manage-users.md:2]: I feel this page it too long... too many articles. 
maybe add a page/child page for "managing passwords" (or something like that)
and a page/child page for all the lockouts (hardening?)

File: content/rs/security/access-control/_index.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/security/access-control/_index.md]: I added a mention of pub/sub here.

File: content/rs/security/access-control/active-active.md (hunk 2 above)
Comment (rrelledge) [reply content/rs/security/access-control/active-active.md:16]: Added another header.

**Review by AlonMagrafta — APPROVED** [review]

**Pull request merged**